bob-common/src/configs/cluster.rs

```diff
@@ -428,62 +418,60 @@ impl Cluster {
 impl Validatable for Cluster {
     fn validate(&self) -> Result<(), String> {
         if self.nodes.is_empty() {
-            let msg = "bob requires at least one node to start".to_string();
-            error!("{}", msg);
-            return Err(msg);
+            error!("bob requires at least one node to start");
```

Comment:
Why duplicate logs here while err will be processed up the stack?

Reply:
Updated. Now errors are printed on the top level.

I checked that at this point logger is not initialized, thus this messages goes nowhere